pkgs/by-name/do/dovecot-fts-flatcurve/package.nix

```
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
 
   configureFlags = [
     "--with-dovecot=${dovecot}/lib/dovecot"
-    "--with-moduledir=$(out)/lib/dovecot"
+    "--with-moduledir=$(out)/etc/dovecot/modules"
```

```suggestion
    "--with-moduledir=${placeholder "out"}/etc/dovecot/modules"
```

And others. `$(out)` is not reliable because it may go through `escapeShellArgs`.

---

Fixed!